README.rst

```diff
@@ -125,20 +127,23 @@ Detailed `installation instructions`_ are available.
 
    ``vagrant plugin install vagrant-service-manager``
 
-4. Download the customized Vagrantfiles provided by the ADB project. These Vagrantfiles will download the ADB and automatically set up provider specific container development environments. They are listed below and more details are available in the `Using Custom Vagrantfiles for Specific Use Cases`_ section of the `Using the Atomic Developer Bundle`_ document and the `Installation document`_.
+4. Download the customized Vagrantfiles provided by the ADB project. These Vagrantfiles
+   will download the ADB and automatically set up provider specific container
+   development environments. They are listed below and more details are available in the `Installation document`_.
 
    To download the ADB and set up provider specific container development environment:
 
    1. Create a directory for the Vagrant box
 
       ``$ mkdir directory && cd directory``
-   
-   2. Download any of the following vagrantfiles, based on your requirements, to download the ADB and use it with host-based tools or via ``vagrant ssh``.
- 
+
+   2. Download any of the following vagrantfiles, based on your requirements, to
+      download the ADB and use it with host-based tools or via ``vagrant ssh``.
```

Comment:
This pair of sentences is awkward.  I believe we should have a simpler statement about select the vagrantfile that configures the environment you need.